### PR TITLE
RMMIS-6345-support-for-persisting-todays-date

### PR DESCRIPTION
**Pull request by kjschulz**
RMMIS-6345 - Added support for persisting today’s date in the application

### CODE REVIEW

File: src/Date.jsx [hunk 1]
```diff
@@ -62,7 +63,12 @@ let DateField = React.createClass({
     this.setState({value});
   },
 
-  componentWillReceiveProps(nextProps) {    
+  componentDidMount() {
+    let dateString = utils.getDateString(this.state.value);
+    this.onChange({target: {value: this.state.value, dateString: dateString}});
```
Comment (mallenscdhhs) [inline src/Date.jsx]: could just write `this.onChange({target: {value: this.state.value, dateString}});`

Comment (mallenscdhhs) [inline src/Date.jsx]: Have you tested this within PE to make sure calling `onChange` within `componentDidMount()` does cause issues?

Comment (kjschulz) [inline src/Date.jsx]: Updated per your suggestion.  Yup, tested...works like a charm!

**Comment by traviscooperscdhhs** [comment]
:+1:

**Pull request merged**